packages/contractkit/src/wallets/rpc-wallet.test.ts

```diff
@@ -0,0 +1,68 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { generateKeys, generateMnemonic } from '@celo/utils/src/account'
+import { normalizeAddressWith0x, privateKeyToAddress } from '@celo/utils/src/address'
+import { RpcWallet } from './rpc-wallet'
+
+testWithGanache('rpc-wallet', (web3) => {
```

Comment:
There are some pretty good wallet tests here that should be pulled into these tests:
https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/src/wallets/local-wallet.test.ts